webapp/channels/src/components/post_view/message_attachments/message_attachment_list.tsx

```diff
@@ -33,32 +33,26 @@ type Props = {
     imagesMetadata?: Record<string, PostImage>;
 }
 
-export default class MessageAttachmentList extends React.PureComponent<Props> {
-    static defaultProps = {
-        imagesMetadata: {},
-    };
-
-    render() {
-        const content = [] as JSX.Element[];
-        this.props.attachments.forEach((attachment, i) => {
-            content.push(
-                <MessageAttachment
-                    attachment={attachment}
-                    postId={this.props.postId}
-                    key={'att_' + i}
-                    options={this.props.options}
-                    imagesMetadata={this.props.imagesMetadata}
-                />,
-            );
-        });
-
-        return (
-            <div
-                id={`messageAttachmentList_${this.props.postId}`}
-                className='attachment__list'
-            >
-                {content}
-            </div>
-        );
-    }
-}
+const MessageAttachmentList = ({
+    imagesMetadata = {},
```

Comment:
To make sure the default is always the same, you can define outside of the component something like `const EMPTY_METADATA: Record<string, PostImage> = {};` and use that here. That way, the default is always the same reference, and you avoid potential re-renders down the line.